packages/trpc/server/routers/viewer/eventTypes/create.handler.ts

```diff
@@ -51,6 +51,7 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
     // Only connecting the current user for non-managed event types and non team event types
     users: isManagedEventType || schedulingType ? undefined : { connect: { id: userId } },
     locations,
+    schedule: scheduleId ? { connect: { id: scheduleId } } : undefined,
```

Comment:
technically can pass a schedule that does not belong to the user but it doesn't pose any security concern

Reply:
Fixed in @supalarry
[fix: dont allow adding scheduleId to event-type if user does not own](https://github.com/calcom/cal.com/pull/15803/commits/2bdb93e0904a77a987368578a64ef0d87396ffef)